web/modules/custom/myroute/myroute.routing.yml

```diff
@@ -0,0 +1,17 @@
+myroute.practicingRoute:
+  path: '/routeit/practice'
+  defaults: 
+    _controller: 'Drupal\myroute\Controller\MyrouteController::practicingRoute'
+    _title: 'Route Content'
+  requirements:
+    _permission: 'access custom page'
+    _role: administrator + 	content_editor
+
```

Comment:
unnecessary extra space

Reply:
There is a 2 line gap.
line 9,10

Is that necessary?

Reply:
Provided for better readability.
Removed as per the suggestion.